cfme/tests/automate/custom_button/__init__.py

```diff
@@ -50,6 +50,40 @@
     "VM_INSTANCE",
 ]
 
+CLASS_MAP = {
```

Comment:
Optional: Thoughts on putting this data in a yaml and placing in the `integration_tests/data` directory?

Reply:
I would like to do this... @john-dupuy still now Custom button rest API lagging with features. Last day I confirm what i want it will take time to in product.... I want to reduce custom button tests run time. I am going to refactor it one day as button and group connection possible with rest. For now, let's put this mapping along with 59OBJ.